### PR TITLE
Updated marchid

### PR DESCRIPTION
**Pull request by Silabs-ArjanB**
Updated marchid according to requested (not yet approved) marchid, see https://github.com/riscv/riscv-isa-manual/pull/630

Signed-off-by: Arjan Bink <Arjan.Bink@silabs.com>

### CODE REVIEW

**Comment by davideschiavone** [comment]
waiting for https://github.com/riscv/riscv-isa-manual/pull/630/commits/221d54b431918618453f79f3c7dfee1d18273c1b to be merged

**Comment by Silabs-ArjanB** [comment]
Hi @davideschiavone https://github.com/riscv/riscv-isa-manual/pull/630/ has been merged now; can you again look at this PR please?

File: docs/user_manual/source/control_status_registers.rst [hunk 1]
```diff
@@ -1031,14 +1031,14 @@ Machine Architecture ID (``marchid``)
...
-Reset Value: 0x0000_0004
+Reset Value: 0x0000_0015
 
 Detailed:
 
 +-------------+-----------+------------------------------------------------------------------------+
 |   Bit #     |   Mode    |   Description                                                          |
 +=============+===========+========================================================================+
-| 31:0        | RO        | Machine Architecture ID of |corev| is 4                                |
+| 31:0        | RO        | Machine Architecture ID of |corev| is 0x15                             |
```
Comment (davideschiavone) [inline docs/user_manual/source/control_status_registers.rst]: I would set this number in degimal format as the RISC-V official repository and the CV32E40P manual. Only the one in the description. The reset value should stay in hex format

**Review by davideschiavone — CHANGES_REQUESTED** [review]

File: docs/user_manual/source/control_status_registers.rst (hunk 1 above)
Comment (Silabs-ArjanB) [reply docs/user_manual/source/control_status_registers.rst]: Hi @davideschiavone It seems that you are looking at an old version of that file. The most recent commit includes the digital representation as well.

Comment (davideschiavone) [reply docs/user_manual/source/control_status_registers.rst]: you are right sorry. pls go ahead with merge

**Pull request merged**